README.md

```diff
@@ -100,7 +120,18 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema converts an ERB into an actual database, and defines how data from one table can grab data from another one. In the models folder, you would need to include code like the following:
+
+class Person < ActiveRecord::Base
+  has_many :wishes
+end   
+
+class Wish < ActiveRecord::Base
+  has_one :person
+end
+
+Every Wish item would have to include an owner_id or wisher_id, which identifies the one person that it belongs to. If you save a particular person to a variable, e.g. Aladdin, you could then see a list of all his wishes with Aladdin.wishes.
+
 ```
```

Comment:
The "other side" of `:has_many` is more commonly `:belongs_to`. This allows the database to more closely represent the nature of the relationship you describe in the last paragraph (using the phrase "belongs to"...). Gotta love that semantic naming!